portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,8 +50,27 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String firstName = (String) entity.getProperty("firstName");
       String lastName = (String) entity.getProperty("lastName");
       String text = (String) entity.getProperty("text");
-      String message = text + " - " + firstName + " " + lastName;
+      Double sentimentScore = (Double) entity.getProperty("sentimentScore");
+      String sentimentScoreStr = String.format("%.2f", sentimentScore);
+      
+      String emojiCode;
+      if (sentimentScore < -0.5) {
+        emojiCode = "0x1F620";
+      } else if (sentimentScore >= -0.5 && sentimentScore < 0) {
```

Comment:
I think you can drop the `sentimentScore >= -0.5` check from this `if` statement? Since you already check `if (sentimentScore < 0.5)` above, you know that's not true, so `sentimentScore >= -0.5` will always be true, right?

In other words, I think this  can be:

```
else if (sentimentScore < 0) {
```

(Same thing for the other `else if` statement below.)

Reply:
oh yeah you're right